kosekaku/Property-pro-liteV2
# Api spec

**What does this PR do?**
- resolve cannot send headers after they are sent to the client 
- add test cases with mocha, chai , and chai-http for integration testing
- fix bugs

**Description of Task to be completed**
- fix bugs
- add testing
- resolve delete property api routes bug

**How should this be manually tested?**
- clone the repo and switch to the branch bug-develop-api-mocha-167270354
- run npm run dev  to start the server
- run npm run test to test results
- run npm run coveralls to see the test output coverage

**Any background context you want to provide?**

    N/A

**What are the relevant pivotal tracker stories?**

    NA
 **Screenshots (if appropriate)**
    N/A
